docs/contributing.md

```diff
@@ -6,51 +6,53 @@ Contributions are welcome and encouraged! We ask only that all contributors foll
 
 Before opening a PR to contribute code you should check that your changes work by running the test suite locally.
 
+```{important}
+:name: dependencies
+We use [pixi](https://pixi.sh/latest/) to manage dependencies, which you'll want to install to get started.
+```
+
+Run tests with the `pixi run --environment test run-tests` command. Some tests require downloading files over the network.
+Use the `run-tests-no-network` task if you want to run tests faster or have no internet access:
+
 ```bash
-mamba env create -f ci/environment.yml
-mamba activate virtualizarr-tests
-pre-commit install
-# git checkout -b new-feature
-python -m pip install -e . --no-deps
-python -m pytest
+# Run all tests
+pixi run --environment test run-tests
+# Skip tests that require a network connection
+pixi run --environment test run-tests-no-network
 ```
 
-You may also add the `--run-network-tests` option, which will run additional tests
-that require downloading files over the network.  Skip this if you want the tests to run
-faster or you have no internet access:
+You can also run tests in other environments:
 
 ```bash
-python -m pytest --run-network-tests
+pixi run --environment min-deps run-tests # Test with the minimal set of dependencies installed
+pixi run --environment upstream run-tests # Test with unreleased versions of upstream libraries
+
 ```
 
 Further, the `pytest-cov` plugin is a test dependency, so you can generate a test
 coverage report locally, if you wish (CI will automatically do so).  Here are some
 examples:
 
 ```bash
-python -m pytest --cov=.                     # Terminal (text) report (--cov=term)
-python -m pytest --cov=. --cov=term-missing  # Terminal report showing missing coverage
-python -m pytest --cov=. --cov=html          # HTML report written to htmlcov/index.html
+pixi run --environment test run-tests-cov              # Terminal report showing missing coverage
+pixi run --environment test run-tests-html-cov         # HTML report written to htmlcov/index.html
 ```
 
-To see all available `pytest` options added by the `pytest-cov` plugin, run
-`python -m pytest -h`, or see the
-[pytest-cov documentation](https://pytest-cov.readthedocs.io/en/latest/readme.html).
-
 ## Contributing documentation
 
 Whilst the CI will build the updated documentation for each PR, it can also be useful to check that the documentation has rendered as expected by building it locally.
 
 ### Build the documentation locally
 
 ```bash
-mamba env create -f ci/doc.yml
-mamba activate virtualizarr-docs
-python -m pip install -e .  # From project's root - needed to generate API docs
-cd docs # From project's root
-rm -rf generated
-make clean
-make html
+pixi install --environment docs
+pixi run build-docs
+```
+Pixi can also be used to serve continuously updating version of the documentation during development at [http://0.0.0.0:8000/](http://0.0.0.0:8000/).
```

Comment:
Oh that's cool. Will that work with #486?

Reply:
yeah, unfortunately they'll have some merge conflicts to resolve but it uses the same `sphinx-autobuild` extension they reference.